packages/synapse-react-client/src/components/HasAccess/HasAccessV2.tsx

```diff
@@ -100,13 +100,6 @@ function AccessIcon(props: { downloadType: FileHandleDownloadTypeEnum }) {
   return <></>
 }
 
-function isFileHandleDownloadTypeUndefined(
-  fileHandleDownloadType: FileHandleDownloadTypeEnum | undefined,
-) {
-  // note, this can't be "if (!downloadType)" since DownloadTypeEnum has a 0 value (which is falsy)
-  return fileHandleDownloadType === undefined
-}
-
```

Comment:
Comment is no longer accurate, so a separate method is not needed -- can just check with `!fileHandleDownloadType` as needed